src/theme/mui.ts

```diff
@@ -160,6 +160,12 @@ const theme = createTheme({
         position: 'absolute',
         top: '5px',
         zIndex: 1, // for firefox
+
+        '&.Mui-error': {
+          paddingLeft: '0',
+          width: '100%',
+          textAlign: 'center',
```

Comment:
Looks like this needs to be `left`.

Reply:
I removed that style as the inputs a slightly wider now